libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -673,14 +681,16 @@ bool AP_DDS_Client::create()
             requests[2] = dwriter_req_id;
 
             if (!uxr_run_session_until_all_status(&session, requestTimeoutMs, requests, status, nRequests)) {
-                GCS_SEND_TEXT(MAV_SEVERITY_ERROR,"XRCE Client: Topic/Pub/Writer session request failure for index '%u'",i);
+                //probably have errors on a different debug level
```

Comment:
Agree, can you figure out how to do this?

Reply:
@Ryanf55 updated the PR..could you take a look ?
Thanks